template.tpl

```diff
@@ -0,0 +1,550 @@
+    "displayName": "",
+    "macrosInSelect": false,
+    "selectItems": [
+      {
+        "value": "Lax",
+        "displayValue": "Lax"
+      },
+      {
+        "value": "None",
+        "displayValue": "None"
+      },
+      {
+        "value": "Strict",
+        "displayValue": "Strict"
+      }
+    ],
+    "simpleValueType": true
+  },
+  {
+    "type": "CHECKBOX",
+    "name": "enableBrowserFeatureDetection",
+    "checkboxText": "Enable Browser Feature Detection",
+    "simpleValueType": true,
+    "help": "By default, Matomo accesses some information from the visitor’s browser like the browser resolution. Some privacy regulations may allow access of such information from the visitor’s device only after having a consent. If this applies to you, you can disable this feature.\n\nTo learn more, check this https://matomo.org/faq/how-to/how-do-i-disable-browser-feature-detection-completely/."
+  },
+  {
+    "type": "PARAM_TABLE",
+    "name": "domains",
+    "displayName": "Domains",
+    "paramTableColumns": [
+      {
+        "param": {
+          "type": "TEXT",
+          "name": "domain",
+          "displayName": "Domain",
+          "simpleValueType": true
+        },
+        "isUnique": false
+      }
+    ],
+    "help": "Used to detect outlinks. Add hostnames or domains to be treated as local. For wildcard subdomains, you can use: \".example.com\" or \"*.example.com\". You can also specify a path along a domain: \"*.example.com/subsite1\".",
+    "editRowTitle": "Edit Domain",
+    "newRowButtonText": "Add Domain",
+    "newRowTitle": "Domain"
+  },
+  {
+    "type": "CHECKBOX",
+    "name": "doNotUseSendBeacon",
+    "checkboxText": "Do not use sendBeacon",
+    "simpleValueType": true,
+    "help": "Uses regular ajax request."
+  },
+  {
+    "type": "TEXT",
+    "name": "userId",
+    "displayName": "User ID",
+    "simpleValueType": true,
+    "textAsList": false,
+    "help": "Sets a User ID to this user (such as an email address or a username)."
+  },
+  {
+    "type": "PARAM_TABLE",
+    "name": "customDimensions",
+    "displayName": "Custom Dimensions",
+    "paramTableColumns": [
+      {
+        "param": {
+          "type": "TEXT",
+          "name": "index",
+          "displayName": "Index",
+          "simpleValueType": true
+        },
+        "isUnique": false
+      },
+      {
+        "param": {
+          "type": "TEXT",
+          "name": "value",
+          "displayName": "Value",
+          "simpleValueType": true
+        },
+        "isUnique": false
+      }
+    ],
+    "newRowTitle": "Custom Dimension",
+    "newRowButtonText": "Add Custom Dimension",
+    "help": "Optionally set one or multiple custom dimensions."
+  }
+]
+
+
+___SANDBOXED_JS_FOR_WEB_TEMPLATE___
+
+// Enter your template code here.
+var log = require('logToConsole');
+var createQueue = require('createQueue');
+var injectScript = require('injectScript');
+log('data =', data);
+
+var onSuccess = () => {
```

Comment:
Are there any compatibility concerns with using arrow functions? According to [caniuse.com](https://caniuse.com/?search=arrow), arrow functions aren't supported by IE or Opera Mini. I don't know if we need to worry about supporting those two browsers.

Reply:
@snake14 we did removed `let` and `const` to make it more compatible, and it makes sense to update here too.
I have updated the code, please check

Reply:
Thank you.